### PR TITLE
:stew: Updates catering menu

### PR DESCRIPTION
**Pull request by jefftriplett**

### CODE REVIEW

File: _pages/catering-menus.md [hunk 1]
```diff
@@ -7,40 +7,163 @@ description: |
...
 ----
 
-## Tutorials: Sunday, September 22 - Lunch
+## Breakfast
 
-### TBD
+### Sunday - Wednesday Breakfasts
 
-----
+- Chilled Orange, Apple & Cranberry Juices
```
Comment (drewbrew) [inline _pages/catering-menus.md]: Inconsistent use of oxford comma vs line 16 above and lines 26-27 below

File: _pages/catering-menus.md [hunk 2]
```diff
@@ -7,40 +7,163 @@ description: |
...
+- Seasonal Fresh Fruit
+- Freshly Baked Breakfast Breads, Pastries, Butter, & Fruit Preserves, Gluten - Free Options available
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Freshly Baked Breakfast Breads, Pastries, Butter, & Fruit Preserves, Gluten Free Options available
```

File: _pages/catering-menus.md [hunk 3]
```diff
@@ -7,40 +7,163 @@ description: |
...
+- Assorted Cereals, Yogurt and Granola
+- Freshly Brewed Coffee, Decaffeinated Coffee, & Selection of Teas
+
+### Thursday - Friday Breakfasts
+- Chilled Orange, Apple & Cranberry Juices
+- Seasonal Fresh Fruit
+Freshly Baked Breakfast Breads, Pastries, Butter, & Fruit Preserves, Gluten - Free Option available
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
Freshly Baked Breakfast Breads, Pastries, Butter, & Fruit Preserves, Gluten Free Option available
```

File: _pages/catering-menus.md [hunk 4]
```diff
@@ -7,40 +7,163 @@ description: |
     Soups up!
 ---
 
-## Continental Breakfast - All Days
+## Afternoon Breaks
 
-- Chilled Orange, Apple, and Cranberry Juices
+- Assorted Granola Bars, GF Options available
+- Assorted Freshly Baked Cookies & Brownies, GF Options available
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Assorted Freshly Baked Cookies & Brownies, Gluten-free Options available
```

File: _pages/catering-menus.md [hunk 5]
```diff
@@ -7,40 +7,163 @@ description: |
     Soups up!
 ---
 
-## Continental Breakfast - All Days
+## Afternoon Breaks
 
-- Chilled Orange, Apple, and Cranberry Juices
+- Assorted Granola Bars, GF Options available
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Assorted Granola Bars, Gluten-free Options available
```

File: _pages/catering-menus.md [hunk 6]
```diff
@@ -7,40 +7,163 @@ description: |
...
+#### Farmers Market-Herb Garlic Roasted Chicken Lunch Buffet
+
+- Assorted Bread Rolls and Butter
+  (gluten-free available)
+- Soup of the Day
+  (vegan/dairy-free available)
+- Garlic Croutons
+  (gluten-free available)
+- Salad Bar
+    - Romaine Lettuce, Spinach, Cucumber, Tomatoes, Mushroom, Carrots, Kalamata Olives
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
  - Romaine Lettuce, Spinach, Cucumber, Tomatoes, Mushroom, Carrots, Kalamata Olives
```

File: _pages/catering-menus.md [hunk 7]
```diff
@@ -7,40 +7,163 @@ description: |
...
+  - Smoked Bacon, Garbonzo Beans, Gorgonzola, Parmesan, Hard Boiled Eggs, Artichokes
+  (non-meat substitute for bacon available; non-dairy cheese available)
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
    (non-meat substitute for bacon available; non-dairy cheese available)
```

File: _pages/catering-menus.md [hunk 8]
```diff
@@ -7,40 +7,163 @@ description: |
...
-## Talks: Tuesday, September 24 - Lunch
+### Talks: Monday, September 23 - Lunch
+
+#### Tuscany Lunch Buffet
 
-### TBD
+- Minestrone Soup
+  (vegan/vegetarian, dairy-free and gluten-free available)
+- Insalata Caprese - Field Greens, Roma Tomatoes, Fresh Mozzarella, Sea Salt, Basil Balsamic Vinaigrette
+- Classic Caesar Salad - Romaine Lettuce, Freshly Baked Croutons, Shaved Parmezan (non-dairy parmesan option, gluten-free croutons available)
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Classic Caesar Salad - Romaine Lettuce, Freshly Baked Croutons, Shaved Parmesan (non-dairy Parmesan option, gluten-free croutons available)
```

File: _pages/catering-menus.md [hunk 9]
```diff
@@ -7,40 +7,163 @@ description: |
...
+- Cavatappi, Roasted Vegetables, Spinach, House Made Tomato Sauce, Olives, Basil, Shaved Parmesan
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Cavatappi, Roasted Vegetables, Spinach, House-made Tomato Sauce, Olives, Basil, Shaved Parmesan
```

File: _pages/catering-menus.md [hunk 10]
```diff
@@ -7,40 +7,163 @@ description: |
...
+  (non-dairy option for parmesan; gluten-free pasta option available)
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
  (non-dairy option for Parmesan; gluten-free pasta option available)
```

File: _pages/catering-menus.md [hunk 11]
```diff
@@ -7,40 +7,163 @@ description: |
...
 ----
 
-## Talks: Wednesday, September 25 - Lunch
+### Talks: Tuesday, September 24 - Lunch
+
+#### South of the Border Lunch Buffet
+
+- Tortilla Soup
+  (dairy-free and meat-free option; gluten-free option available)
+- Southwestern Salad - Romaine Lettuce, Pepperjack, Black Beans, Roasted Corn, Tortilla Strips, Honey Chipotle Ranch (dairy-free cheese available)
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Southwestern Salad - Romaine Lettuce, Pepper Jack, Black Beans, Roasted Corn, Tortilla Strips, Honey Chipotle Ranch (dairy-free cheese available)
```

File: _pages/catering-menus.md [hunk 12]
```diff
@@ -7,40 +7,163 @@ description: |
...
 
-## Sprints: Thursday, September 26 - Friday, September 27 - Lunch
+### Sprints: Thursday, September 26
 
-### TBD
+#### Sandwich Shoppe Lunch Buffet
+- Soup of the Day, Vegan and Gluten Free available
+- Baby Spinach Salad: Goat Cheese, Walnuts, Dried Cherries, and Balsamic Vinaigrette
+- Seasonal Fresh Fruit
+- Baja Chicken Wrap: Flour Tortilla, Peppers, Onions, Guacamole, Pepperjack, Shredded Lettuce, - Chipotle Aioli
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Baja Chicken Wrap: Flour Tortilla, Peppers, Onions, Guacamole, Pepper Jack, Shredded Lettuce, - Chipotle Aioli
```

File: _pages/catering-menus.md [hunk 13]
```diff
@@ -7,40 +7,163 @@ description: |
...
+- Italiano: Salami, Ham, Capicola, Provolone, Olives, Pepperoncini, Tomatoes, Lettuce, - Italian Vinaigrette, Hoagie
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Italiano: Salami, Ham, Capicola, Provolone, Olives, Pepperoncini, Tomatoes, Lettuce, Italian Vinaigrette, Hoagie
```

File: _pages/catering-menus.md [hunk 14]
```diff
@@ -7,40 +7,163 @@ description: |
...
 ----
 
-## Tutorials: Sunday, September 22 - Lunch
+## Breakfast
 
-### TBD
+### Sunday - Wednesday Breakfasts
 
-----
+- Chilled Orange, Apple & Cranberry Juices
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Chilled Orange, Apple, & Cranberry Juices
```

File: _pages/catering-menus.md [hunk 15]
```diff
@@ -7,40 +7,163 @@ description: |
...
+- Seasonal Fresh Fruit
+- Freshly Baked Breakfast Breads, Pastries, Butter, & Fruit Preserves, Gluten-free Options available
+- Assorted Cereals, Yogurt and Granola
+- Freshly Brewed Coffee, Decaffeinated Coffee, & Selection of Teas
+
+### Thursday - Friday Breakfasts
+- Chilled Orange, Apple & Cranberry Juices
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Chilled Orange, Apple, & Cranberry Juices
```

File: _pages/catering-menus.md [hunk 16]
```diff
@@ -7,40 +7,163 @@ description: |
...
 
-### TBD
+#### Sandwich Shoppe Lunch Buffet
+- Soup of the Day, Vegan and Gluten Free available
+- Baby Spinach Salad: Goat Cheese, Walnuts, Dried Cherries, and Balsamic Vinaigrette
+- Seasonal Fresh Fruit
+- Baja Chicken Wrap: Flour Tortilla, Peppers, Onions, Guacamole, Pepper Jack, Shredded Lettuce, - Chipotle Aioli
+- Italiano: Salami, Ham, Capicola, Provolone, Olives, Pepperoncini, Tomatoes, Lettuce, Italian Vinaigrette, Hoagie
+- Smoked Turkey Breast: Brie, Green Apple, Arugula, Tomato, Dijon Mayo, Brioche
+- Caprese Wrap: Flour Tortilla, Heirloom Tomato, Fresh Mozzarella, Arugula, Balsamic Basil - Aioli
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Caprese Wrap: Flour Tortilla, Heirloom Tomato, Fresh Mozzarella, Arugula, Balsamic Basil, Aioli
```

File: _pages/catering-menus.md (hunk 1 above)
Comment (jefftriplett) [reply _pages/catering-menus.md]: updated, thanks

File: _pages/catering-menus.md [hunk 17]
```diff
@@ -7,40 +7,163 @@ description: |
...
 
-## Sprints: Thursday, September 26 - Friday, September 27 - Lunch
+### Sprints: Thursday, September 26
 
-### TBD
+#### Sandwich Shoppe Lunch Buffet
+- Soup of the Day, Vegan and Gluten Free available
+- Baby Spinach Salad: Goat Cheese, Walnuts, Dried Cherries, and Balsamic Vinaigrette
+- Seasonal Fresh Fruit
+- Baja Chicken Wrap: Flour Tortilla, Peppers, Onions, Guacamole, Pepper Jack, Shredded Lettuce, - Chipotle Aioli
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Baja Chicken Wrap: Flour Tortilla, Peppers, Onions, Guacamole, Pepper Jack, Shredded Lettuce, Chipotle Aioli
```

File: _pages/catering-menus.md [hunk 18]
```diff
@@ -7,40 +7,163 @@ description: |
...
+- Coffee, Decaffeinated Coffee, Hot Tea
 
 ----
 
-## Sprints: Thursday, September 26 - Friday, September 27 - Lunch
+### Sprints: Thursday, September 26
 
-### TBD
+#### Sandwich Shoppe Lunch Buffet
+- Soup of the Day, Vegan and Gluten Free available
```
Comment (drewbrew) [inline _pages/catering-menus.md]:
```suggestion
- Soup of the Day, Vegan and Gluten-free available
```

**Pull request merged**